pkg/gotohelm/transpiler_test.go

```diff
@@ -285,11 +342,26 @@ func NewGoRunner(root string) *GoRunner {
 		"GO111MODULE=on",
 	)
 
-	return &GoRunner{
+	runner := &GoRunner{
 		cmd:      cmd,
 		inputCh:  make(chan *helmette.Dot),
 		outputCh: make(chan map[string]any),
 	}
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
NIT: Could the context be based on context created in https://github.com/redpanda-data/helm-charts/blob/102a2faf6294c9092a9ffc9f39c05a204da66b20/pkg/gotohelm/transpiler_test.go#L143 ?

Reply:
This is a weird case. It's done this way to ensure that any errors get appropriately piped to the `testing.T` so there aren't silent failures. You could plumb another context in but it would be doing the same job as the `t.Cleanup` call but without the correct error propagation.